DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Delegates/DelegateGroups/GroupCourseRemoveViewModelTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups;
+    using NUnit.Framework;
+    using FluentAssertions;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq;
+
+    public class GroupCourseRemoveViewModelTests
+    {
+        [Test]
+        public void GroupCourseRemoveViewModel_Constructor_CorrectlySetsProperties()

```

Comment:
Indeed it is fairly basic. It can be argued either way really, all depends on the project in question and the underlying requirements. I'm in favour of the test given there are expected parameters to pass in and an expected state for the resulting model. A change to the contructor could lead to an undesired (and potentially an unnoticable) state of the resulting model. 
But I appreciate the "minor severity" of such a change vs the overhead of managing such extensive coverage. 
Given the context and use I guess there are other mitigations such as the accessibility tests which check the contents of the `<h1>` tag which should include the `courseName`.

Reply:
Yeah that's fair. It's nice to have, so I think we can keep this here. Just good to be aware that there aren't any other similar tests to encounter, and the value of writing more in future might be called into question.